rfcs/013-yaml-enum-representation.md

```diff
@@ -0,0 +1,52 @@
+# RFC-013: Enum representation in YAML encoding
+
+[![Issue](https://img.shields.io/github/issues/detail/state/kamu-data/open-data-fabric/62?label=Issue)](https://github.com/kamu-data/open-data-fabric/issues/70)
+[![Spec PR](https://img.shields.io/github/pulls/detail/state/kamu-data/open-data-fabric/63?label=PR)](https://github.com/kamu-data/open-data-fabric/pull/63)
+
+**Start Date**: 2023-12-14
+
+**Authors**:
+- [Sergii Mikhtoniuk](mailto:sergii.mikhtoniuk@kamu.dev), [Kamu](https://kamu.dev)
+
+**Compatibility**:
+- [ ] Backwards-compatible
+- [ ] Forwards-compatible
+
+## Summary
+Prpoposes to use `kind: PascalCaseVariantName` convention to standardize how enum variants are differentiated in YAML / JSON serialization format.
```

Comment:
Prpopo, hm :-)

Reply:
I really need to install some spell checker :)